doc/contribute/modifying_contributions.rst

```diff
@@ -0,0 +1,70 @@
+.. _modifying_contributions:
+
+Modifying Contributions made by other developers
+************************************************
+
+Scenarios
+#########
+
+In general, Zephyr contributors and collaborators are encouraged to assist
+as reviewers in pull requests, so that patches may be approved and merged
+to Zephyr's main branch as part of the original pull requests. The authors
+of the pull requests are responsible for amending their original commits
+following the review process.
+
+There are occasions, however, when a contributor might need to modify patches
+included in pull requests that are submitted by other Zephyr contributors.
+For instance, this is the case when:
+
+* a developer picks commits submitted by other contributors into their own
+  pull requests in order to:
+
+  * integrate useful content which is part of a stale pull request, or
+  * get content merged to the project's main branch as part of a larger
+    patch
+
+* a developer pushes onto a branch or pull request opened by another
+  contributor in order to:
+
+  * assist in revising pull requests and get the patches merged
+    to the project's main branch
+  * finalize review in stale pull requests
+
+
+Accepted policies
+#################
+
+A developer who intends to picking and modifying patches sent by
+another contributor shall:
+
+* clarify in their pull request the reason for doing that, instead
+  of assisisting in getting the patches merged in their original
+  pull request, and
+* invite the original author of the patches in their pull request review.
+
+A developer who intends to force-push onto a branch or pull request of
+another Zephyr contributor shall clarify in the pull request the reason
+for pushing and for modifying the existing patches (e.g. stating that it
+is done for assisting in the pull request review when the pull
+request author is not able to do so).
+
+.. note::
+  Developers should try to limit the above practice to pull requests identified
+  as *stale*. Read about how to identify pull requests as stale in
+  :ref:`development processes and tools <dev-environment-and-tools>`
+
+If the original patches are substantially modified, the developer can either:
+
+* (preferably) reach out to the original author and request them to
+  acknowledge that the modified patches may be merged while having
+  the original sign-off line and author identity, or
+* submit the modified patches as their *own* work (i.e. with their
+  *own* sign-off line and author identity). In this case, the developer
+  may want to identify in the commit message(s) the original source the
+  submitted work is based on (mentioning, for example, the original PR
+  number).
+
+.. note::
+  Contributors should be un-checking the box *“Allow Edits By Maintainers"*
```

Comment:
```suggestion
  Contributors should be uncheck the box *“Allow Edits By Maintainers"*
```

Reply:
thanks. I suppose it should be "should uncheck"